### PR TITLE
Added a build stage that pushes to main when the build was triggered …

### PR DESCRIPTION
**Pull request by GarondEisenfaust**
…manually

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 pipeline {
-    agent{
-        node{
+    agent {
+        node {
```
Comment (nemampojma5) [inline Jenkinsfile:3]: mhm

File: Jenkinsfile [hunk 2]
```diff
@@ -33,5 +33,12 @@ pipeline {
                 sh 'mv ./web-build/* /srv/Frontend/'
             }
         }
+        stage('PushToMain') {
+            when { allOf { branch 'dev'; triggeredBy 'UserIdCause' } }
+            steps {
+                echo 'Pushing to main'
+                sh 'git push origin main'
+            }
+        }
```
Comment (nemampojma5) [inline Jenkinsfile:42]: very nice, such wow.

**Review by nemampojma5 — APPROVED** [review]
very nice

File: Jenkinsfile (hunk 1 above)
Comment (GarondEisenfaust) [reply Jenkinsfile:3]: why?:(

Comment (GarondEisenfaust) [reply Jenkinsfile:3]: :'(

**Pull request merged**